packages/medusa/src/loaders/plugins.js

```diff
@@ -104,6 +104,30 @@ async function runLoaders(pluginDetails, container) {
 
 function registerMedusaApi(pluginDetails, container) {
   registerMedusaMiddleware(pluginDetails, container)
+  registerStrategies(pluginDetails, container)
+}
+
+function registerStrategies(pluginDetails, container) {
+  let module
+  try {
+    module =
+      require(`${pluginDetails.resolve}/strategies/tax-calculation`).default
+  } catch (err) {
```

Comment:
nit: Makes sense, since we are only allowing for tax strategies at the moment, but still rather specific. Would it make sense for us to take a more general approach from the get-go or what are your thoughts around this?

Reply:
Thinking we could do something similar to how we register services.

Reply:
Yep - I am thinking that it might also be relevant to have these registered without them being core strategies so it makes a lot of sense; would it be okay to add in a separate PR where we add extended strat support?

Reply:
Sounds great!